lib/fork.js

```diff
@@ -36,7 +37,15 @@ module.exports = (file, opts, execArgv) => {
 		} : false
 	}, opts);
 
-	const args = [JSON.stringify(opts), opts.color ? '--color' : '--no-color'];
+	const args = [JSON.stringify(opts)];
+
+	if (hasFlag('--color')) {
+		args.push(`--color=${opts.color}`);
+	}
+
+	if (hasFlag('--no-color')) {
+		args.push(`--no-color`);
```

Comment:
Although XO doesn't complain, I'd have expected a regular string here, not a templated one.

Reply:
Oh, escaped my eyes.